### DISCUSSION TITLE
What is the Schema of HttpEndpoint used for? Is it the example documentation for Swagger?

### DISCUSSION THREAD

**Discussion by Jauhuei** [elsa-workflows/elsa-core#4422] [Q&A]
What is the Schema of HttpEndpoint used for? Is it the example documentation for Swagger?

**Comment by sfmskywalker** [comment]
Hello @Jauhuei,

Thank you for bringing this to our attention. The Schema of the HttpEndpoint is currently only used to generate intellisense for its output value. However, the idea of using it for generating OpenAPI documentation with Swagger is intriguing and holds potential.

While this feature isn't in the roadmap for Elsa 2, we're always looking for ways to enhance Elsa's capabilities. I'll discuss this with the team, and we might consider it for Elsa 3 or future releases.

Thank you for your valuable input!
